proteus/mprans/RANS2P.h

```diff
@@ -1676,42 +1686,49 @@ namespace proteus
       inline
         void calculateSubgridErrorDerivatives_tauRes(const double& tau_p,
                                                      const double& tau_v,
-                                                     const double dpdeResidualP_du[nDOF_trial_element],
-                                                     const double dpdeResidualP_dv[nDOF_trial_element],
-                                                     const double dpdeResidualP_dw[nDOF_trial_element],
+                                                     const double dpdeResidualP_du[nDOF_v_trial_element],
+                                                     const double dpdeResidualP_dv[nDOF_v_trial_element],
+                                                     const double dpdeResidualP_dw[nDOF_v_trial_element],
```

Comment:
nDOF_trial_element is for the pressure space and nDOF_v_trial_element is for the velocity space.  It gets a little complicated in the vms stuff, particularly the derivatives, because you can differente pressure-related quantities w.r.t velocity.